src/App.js

```diff
@@ -1,11 +1,16 @@
+/* eslint-disable prefer-destructuring */
 import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faCoffee } from '@fortawesome/free-solid-svg-icons'
 import logo from './logo.svg';
 import './App.css';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.onGetData = this.onGetData.bind(this);
+    
+    library.add(faCoffee);
```

Comment:
Hmm interesting. I dunno if I've seen it done this way before. i wonder if doing it this way is necessary. this is the way I've used FA with react:

```
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faCoffee } from '@fortawesome/free-solid-svg-icons';

Your <FontAwesomeIcon icon={faCoffee} /> is hot and ready!
```

But if you prefer this way I'm fine with it either way